app/models/publishing_api_event.rb

```diff
@@ -0,0 +1,6 @@
+class PublishingApiEvent < ApplicationRecord
+  has_many :dimensions_items, class_name: 'Dimensions::Item'
```

Comment:
I'm a bit confused about this relationship, At first glance I would expect the one-to-many to be the other way around.

Reply:
Yes, but if for example the message is for a guide creation, we will have many items (one per path), and all the items are linked to the same message. 
I hope this makes sense. Let me know if you need further clarification.

Reply:
Yes, but if for example the message is for a guide creation, we will have many items (one per path), and all the items are linked to the same message. 
I hope this makes sense. Let me know if you need further clarification.

Reply:
That makes sense